Prevent 'Aborted connection' warnings from the DoctrineConnectionHealthCheck

pablothedude opened the pull request:
Aborted connection warnings where spawned into the `messages` log on every call to the health endpoint. Many sleeping mysql tasks remained unclosed.

To resolve this issue, the DoctrineConnectionHealthCheck needed to be changed. The previously used `execute` statement turned out to cause the problem. Using the `query` method solved the problem.

MKodde committed:
Update CHANGELOG

thijskh commented:
In my test, does not seem to have the desired effect.

tvdijen commented:
Nope, indeed still loads of aborted db-connections from engine, middleware and gateway:
```
Sep 11 20:14:23 database-1 mysqld: 2018-09-11 20:14:23 1702 [Warning] Aborted connection 1702 to db: 'engineblock' user: 'engineblock' host: '10.2.16.3' (Got an error reading communication packets)
Sep 11 20:14:51 database-1 mysqld: 2018-09-11 20:14:51 1703 [Warning] Aborted connection 1703 to db: 'gateway' user: 'gateway' host: '10.2.16.3' (Got an error reading communication packets)
Sep 11 20:15:19 database-1 mysqld: 2018-09-11 20:15:19 1705 [Warning] Aborted connection 1705 to db: 'middleware' user: 'middleware' host: '10.2.16.2' (Got an error reading communication packets)
```

Commit:
Prevent sleeping connections

Prevent sleeping connections in the database which get aborted
eventually. This in order to prevent growing logs.

pablothedude commented:
We've tested the issue on the test environment and could reproduce the problem there. The problem was that the wrong method was used and therefore the connections were not properly closed. This fix should now address this  issue.

thijskh commented:
Yes, works better now!

MKodde commented:
@pablothedude, would you be so kind to install this change in:
* Stepup-*
* SP Dashboard
* User lifcecycle
* Engine (even tho this change was already fixed in the overloaded Checker)

See: https://www.pivotaltracker.com/story/show/158906723
